packages/react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -950,13 +950,14 @@ class DropdownInternal extends React.Component<IDropdownInternalProps, IDropdown
       return;
     }
 
-    // hasFocus tracks whether the root element has focus so always update the state.
-    this.setState({ hasFocus: false });
-
     if (this.state.isOpen) {
-      // Do not onBlur when the callout is opened
+      // Do not call onBlur or update focus state when the callout is opened
       return;
     }
+
+    // hasFocus tracks whether the root element or dropdown has focus so always update the state.
```

Comment:
Sorry to be nitpicky but I took the "always" in the comment to mean "always update even if open." Thinking about it more, maybe it would make more sense at this point to just delete this comment, and update the doc comment for hasFocus to say "Whether the root element or dropdown has focus" (it seems like the purpose of that state may have changed over time?).

Reply:
Sure! And yeah, I'm not sure what the original purpose of hasFocus was, but right now it seems like it's only used to handle openOnFocus.